sql/core/src/test/resources/sql-tests/inputs/describe.sql

```diff
@@ -97,3 +97,14 @@ DROP VIEW temp_v;
 DROP VIEW temp_Data_Source_View;
 
 DROP VIEW v;
+
+-- Show column default values
+CREATE TABLE d (a STRING, b INT DEFAULT 42) USING parquet COMMENT 'table_comment';
```

Comment:
Since this is a test case for DEFAULT value, please add a default value for column `a`, too.